src/main/java/com/redislabs/RGHibernate.java

```diff
@@ -17,38 +20,55 @@
 import org.hibernate.boot.registry.StandardServiceRegistry;
 import org.hibernate.boot.registry.StandardServiceRegistryBuilder;
 
+import gears.GearsBuilder;
+
 
 public class RGHibernate implements Closeable, Serializable {
 
   private static final long serialVersionUID = 1L;
 
-  private String[] args;
+  private static final Map<String, RGHibernate> hibernateConnections = new HashMap<>();
+  
+  public static RGHibernate getOrCreate(String name) {
+    RGHibernate ret = hibernateConnections.get(name);
+    if(ret == null) {
+      ret = new RGHibernate(name);
+    }
+    hibernateConnections.put(ret.name, ret);
+    return ret;
```

Comment:
```suggestion
    RGHibernate ret = hibernateConnections.get(name);
    if(ret == null) {
      ret = new RGHibernate(name);
      hibernateConnections.put(ret.name, ret);
    }
    return ret;
```

Reply:
I want to add it only if it's new, otherwise it's already there.

Reply:
That's exactly what I'm doing here...

Reply:
Ohh you are right, thanks.